### DISCUSSION TITLE
000

### DISCUSSION THREAD

**Discussion by giscus** [zhiyu1998/rrorange-and-friends-weekly#1] [General]
# 000

《RrOrange和朋友们的周刊》： 分享日常冲浪互联网看到好玩的网站、app应用、资源分享、效率软件工具集等

http://localhost:5173/posts/2024/000

**Comment by zhiyu1998** [comment]
test comment

**Comment by loveliveao** [comment]
网站做的也挺好，还支持RSS，以前用过kindle+inoreader订阅一些网站的RSS，也有四五年没用了。越来越跟不上世界的脚步了
